New/src/content/en/events/_index.yaml

```diff
@@ -0,0 +1,71 @@
+project_path: /web/_project.yaml
+book_path: /web/_book.yaml
+landing_page:
+  custom_css_path: /web/events/events.css
+
+  rows:
+
+  - heading: Attend A Developer Event And Get Hacking
+    classname: devsite-landing-row-67
+    items:
+    - description: >
+        Events are run by local Google Developer Groups or other
+        developer meetups. Each event is a little different, but generally, an
+        event can include a tech talk, series of code labs, and time for
+        personal coding.
+
+  - items:
+    - custom_html: <hr>
+
+  - heading: Find Events Near You
+    classname: find-events
+    items:
+    - description: >
+        <p>
+          Level up your skills at a local event. Events consist of code labs
+          and talks by leading domain experts. Find one in your area or start
+          your own!
+        </p>
+
+        <p>
+          If you're not able to find a nearby event don't worry! All the
+          materials are available so you can organize your own event or
+          work through the content at home. <a href="#featured-events">See the
+          featured events section for more details</a>.
+        </p>
+
+    - custom_html: >
+        <div class="doc-iframe-container">
```

Comment:
Is this the way the DevSite team told you to do this, or did they say framebox? 

It looks like it just creates an iframe that uses the same domain, so I'm not sure it'll work. If it's a framebox, there's a slightly different syntax.